ui/src/app/machines/views/MachineDetails/MachineTests/MachineTests.tsx

```diff
@@ -95,13 +108,17 @@ const MachineTests = (): JSX.Element => {
               ]) => {
                 const { model, name, serial } =
                   scriptResults[0]?.parameters?.storage?.value || {};
+                let title = name ? `/dev/${name}` : null;
+                if (name && model && serial) {
```

Comment:
Is it possible to have a model but not a serial number? Might be safer to evaluate them separately.

Reply:
Good thinking.